File: core/src/main/java/org/mifos/mobilewallet/core/data/paymenthub/entity/PartyIdInfo.java
package org.mifos.mobilewallet.core.data.paymenthub.entity;

import com.google.gson.annotations.SerializedName;

public class PartyIdInfo {

    @SerializedName("partyIdType")
    String partyIdType;

    @SerializedName(("partyIdentifier"))
    String partyIdentifier;

    @SerializedName("partySubIdOrType")
    String partySubIdOrType;

    @SerializedName("fspId")
    String fspId;
}

File: core/src/main/java/org/mifos/mobilewallet/core/data/paymenthub/entity/PartyIdInfo.kt
package org.mifos.mobilewallet.core.data.paymenthub.entity

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize

@Parcelize
class PartyIdInfo: Parcelable {

    @SerializedName("partyIdType")
    var partyIdType: String? = null

    @SerializedName("partyIdentifier")
    var partyIdentifier: String? = null

    @SerializedName("partySubIdOrType")
    var partySubIdOrType: String? = null
}
